pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovsflows
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/querier"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform/common"
+)
+
+// Response is the response struct of ovsflows command.
+type Response struct {
+	Flow string `json:"flow,omitempty" yaml:"flow,omitempty"`
+}
+
+func getAllFlows(aq querier.AgentQuerier) ([]Response, error) {
+	resps := []Response{}
+	flowStrs, err := aq.GetOfctlClient().DumpFlows()
+	if err != nil {
+		klog.Errorf("Failed to dump flows: %v", err)
+		return nil, err
+	}
+	for _, s := range flowStrs {
+		resps = append(resps, Response{s})
+	}
+	return resps, nil
+}
+
+func getPodFlows(aq querier.AgentQuerier, podName, namespace string) ([]Response, error) {
+	intf, ok := aq.GetInterfaceStore().GetContainerInterface(podName, namespace)
+	if !ok {
+		return nil, nil
+	}
+
+	flows := aq.GetOpenflowClient().GetPodFlows(intf.InterfaceName)
+	resps := []Response{}
+	for _, f := range flows {
+		flowStrs, err := aq.GetOfctlClient().DumpFlows(f.MatchString())
+		if err != nil {
+			klog.Errorf("Failed to dump flows: %v", err)
+			return nil, err
+		}
+		for _, s := range flowStrs {
+			resps = append(resps, Response{s})
+		}
+	}
+	return resps, nil
+}
+
+// HandleFunc returns the function which can handle queries issued by the pod-interface command,
```

Comment:
"the pod-interface command", copy-paste error.

Reply:
Thanks. Fixed.